README.asciidoc

```diff
@@ -15,7 +15,7 @@ TL;DR
 
 {website}
 
-*Vim inspired* -- *Faster as in fewer keystrokes* --
+*Modal edition* -- *Faster as in fewer keystrokes* --
```

Comment:
While "edition" is an English word, it's usually in the context of a published book. Perhaps "editing" or "editor" would be more appropriate here?

Reply:
I wrote "modal editor" on the site too, fixed.